tests/cli/backup/test_backup_command.py

```diff
@@ -63,8 +63,8 @@ def test_backup_flow_cells_when_dsmc_is_running(
     # THEN exits without any errors
     assert result.exit_code == EXIT_SUCCESS
 
-    # THEN communicate Dsmc process is already running
-    assert "A Dsmc process is already running" in caplog.text
+    # THEN communicate too many Dsmc processes are already running
+    assert "Too many Dsmc processes are already running" in caplog.text
```

Comment:
previously it was one, now it is three processes that are running at the same time. Should we assert that at least three processes are running in this test? if that is possible

Reply:
Very hard since it is an external dependency started as a sub-process - I do think it is worth the effort.